src/test/resources/refaster/NestedPreconditionsRecipe.java

```diff
@@ -96,17 +96,16 @@ public J visitNewClass(J.NewClass elem, ExecutionContext ctx) {
 
         };
         return Preconditions.check(
```

Comment:
I'm just being really nitpicky but wouldn't it be great if this would be a check of `and(common, or(and(hashmap related), and(linkedhashmap related))`?

Reply:
Done!